ui/src/app/stores/application/application.effects.ts

```diff
@@ -25,27 +25,28 @@ import { AppInfoModel } from '../../models/appInfo.model';
 export class ApplicationEffects {
   constructor(private actions: Actions, private appInfoService: AppInfoService) {}
 
-  @Effect({ dispatch: true })
```

Comment:
The dispatch true is not necessary anymore?

Reply:
No, only if it does not dispatch